### DISCUSSION TITLE
fluent-option value

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2455] [General]
The question is strange. The FluentOption are managed by InternalListContext. 
Reading the code I see that

```html
<fluent-option @ref=Element
id="@Id"
...
value="@Value"
...
>

@ChildContent
</fluent-option>
```

Why write the value for html? The value is saved in the class and the conente in `@ChildContent`.

best regards

**Comment by franklupo** [comment]
if it's not written is it a problem?

**Reply by vnbaaij** [reply]
I think the web component needs it

**Reply by franklupo** [reply]
I'm doing an experiment for list components with TOption and TValue where TValue is not just string like now but can be in any object.
